src/Estimators/EstimatorManagerInput.h

```diff
@@ -82,18 +82,6 @@ class EstimatorManagerInput
   EstimatorInputs estimator_inputs_;
   ScalarEstimatorInputs scalar_estimator_inputs_;
 
```

Comment:
Seems clear these should be simplified to
```c++
  template<typename T>
  void appendEstimatorInput(XMLNodePtr node)
  {
    estimator_inputs_.emplace_back(std::in_place_type<T>, node);
  }
```

Reply:
Done.

Reply:
Done.